src/flows/braze/update_flow.py

```diff
@@ -359,7 +369,9 @@ def mask_email_domain_outside_production(rows: List[Dict], email_column='EMAIL')
     return rows
 
 
-with Flow(FLOW_NAME, executor=LocalDaskExecutor(), result=get_s3_result()) as flow:
+with Flow(FLOW_NAME, schedule=schedule, executor=LocalDaskExecutor(), result=get_s3_result()) as flow:
+    dbt_run_result = dbt.DbtCloudRunJob()(cause=FLOW_NAME, job_id=DBT_CLOUD_BRAZE_JOB_ID, wait_for_job_run_completion=True)
```

Comment:
On line 391, on the `PocketSnowflakeQuery`, you'll want to add:
```python
.set_upstream(
        dbt_run_result,  # Run the Snowflake query after the Dbt job finishes
    )
```
(See `create_email_aliases` below for an example.)

Reply:
Done